app/graphql/mutations/create_dynamic_field.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+class Mutations::CreateDynamicField < Mutations::BaseMutation
+  argument :string_key, ID, required: true
+  argument :display_label, String, required: true
+  argument :sort_order, Integer, required: true
```

Comment:
Related: Please see my comment about sort_order in `Mutations::CreateDynamicFieldGroup`.